#3302 make middle mouse close flowgraphs

sidkapoor97 opened the pull request:
References #3302 
I have added the changes, the only issue is right now the graph closes just on left clicking the label_button. I could not figure out the keypress for middle mouse. I used [https://gitlab.gnome.org/GNOME/gtk/-/blob/3.24.14/gdk/keyname-table.h](url) as reference, any help would be really appreciated. Thanks for the advice @dkozel  i have tested the change.

sidkapoor97 commented:
Also for some reason the Block tree panel isn't visible, I couldn't figure out why so

mormj commented:
Thank you for tacking this feature - as you mentioned it looks like there are still some issues:

Normal GRC, the tabs look like this:
![image](https://user-images.githubusercontent.com/34754695/79867052-b6ef3e00-83ab-11ea-9690-ce026d7c312e.png)

But after the PR, the tabs look like:
![image](https://user-images.githubusercontent.com/34754695/79867133-d4240c80-83ab-11ea-9dfb-f71bcfa7489b.png)

So we have also lost the filename, and it looks like a button to close it.  I have no idea how to implement it, but I was hoping this feature would look more like browser tabs where the middle mouse click closes the tab.  

The block tree panel shows up fine for me.  Perhaps that has just been dragged all the way to the right - I've gotten stuck in that state before, but you should be able to grab it and drag it back.

sidkapoor97 commented:
@mormj so I changed the label to a button hence the outline. I have removed the outline, I forgot that to do before, the filename is because of the `set_markup()` function not running for the `label_button` as I couldn't figure it how. Any help would be appreciated

sidkapoor97 commented:
@mormj I have added the markup and now it's working fine, the only issue is middle mouse click where in any help would be appreciated

mormj commented:
@sidkapoor97 - I don't have the answers as to how to make the middle mouse click work.  Please let us know if you were able to figure it out.

mormj commented:
@sidkapoor97 - I'm going to close this PR for now, please open it back up if you are able to get this feature working.  Thanks!
